main.go

```diff
@@ -69,12 +69,13 @@ func main() {
 	}
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:             scheme,
-		MetricsBindAddress: metricsAddr,
-		Port:               9443,
-		LeaderElection:     enableLeaderElection,
-		LeaderElectionID:   "085c249a.redhat.io",
-		Namespace:          watchNamespace,
+		Scheme:                     scheme,
+		MetricsBindAddress:         metricsAddr,
+		Port:                       9443,
+		LeaderElection:             enableLeaderElection,
+		LeaderElectionID:           "085c249a.redhat.io",
+		LeaderElectionResourceLock: "configmaps",
+		Namespace:                  watchNamespace,
```

Comment:
I personally don't like namespace-scoped operators. This in particular has no reason to be namespaces because the CRD should be cluster-scoped has it is governing a cluster-wide aspect.

Reply:
i agree. this is something that I have been meaning to get to and will add for a future release